packages/ckeditor5-list/tests/list/utils.js

```diff
@@ -444,10 +449,31 @@ describe( 'utils', () => {
 			);
 
 			expect( getSiblingNodes( document.selection.getFirstPosition(), 'forward' ) ).to.deep.equal( [
+				document.getRoot().getChild( 2 ),
 				document.getRoot().getChild( 3 ),
 				document.getRoot().getChild( 4 )
 			] );
 		} );
+
+		it( 'should return only list items from block quoted list when selection is inside (direction="forward")', () => {
```

Comment:
I'd like you change the name for this test. This test is not about block quote only, it is about returning items that are contained in a different element. My proposal: "should return only list items that are inside the same parent element". Or "...the same container".